tests/Unit/Domain.LinnApps.Tests/PurchaseOrderServiceTests/WhenUpdatingPurchaseOrderOverrideValueQtyFields.cs

```diff
@@ -392,5 +388,35 @@ public void ShouldUpdateMiniOrderFields()
             this.miniOrder.OrderTotal.Should().Be(19862.33m);
             this.miniOrder.BaseOrderTotal.Should().Be(24827.91m);
         }
+
+        [Test]
+        public void ShouldUpdateDeliveryTotalFields()
+        {
+            this.current.OrderNumber.Should().Be(600179);
+            this.current.Remarks.Should().Be("updated remarks");
+
+            var firstDetail = this.current.Details.First();
+
+            var delivery = firstDetail.PurchaseDeliveries.First();
+
+            // order unit price currency, our unit price currency, our delivery qty
+            delivery.OrderUnitPriceCurrency.Should().Be(120m);
```

Comment:
this test updated this to be 200.22 on the detail - Why are you testing that its 120 here?